src/app/mat-select-search/mat-select-search.component.ts

```diff
@@ -38,6 +39,25 @@ import { MatSelectSearchClearDirective } from './mat-select-search-clear.directi
 /** The max height of the select's overlay panel. */
 const SELECT_PANEL_MAX_HEIGHT = 256;
 
+/** InjectionToken that can be used to specify global options. */
+export const MATSELECTSEARCH_GLOBAL_OPTIONS = new InjectionToken<MatSelectSearchOptions>('MATSELECTSEARCH_GLOBAL_OPTIONS');
+
+/** Configurable options for MatSelectSearch. */
+export interface MatSelectSearchOptions {
+  closeIcon?: string;
```

Comment:
please copy the docs to each of the options

Reply:
Mentioned above

Reply:
Maybe we can simply add a note like „see the documentation of the corresponding @Input() properties of MatSelectSearchComponent„

Reply:
(This is optional, but would help stricter type checks, and ut ensures the properties actually exist)